tests/index.js

```diff
@@ -54,9 +53,9 @@ tap.test('should validate querystring parameters', async t => {
         }
     }, (req, res) => res.send(req.query))
     await fastify.listen({ port: 5000 })
```

Comment:
```suggestion
```

Reply:
there is no need to listen now.

Reply:
@mcollina Thanks, I actually wasn't aware of `fastify.inject()` (very useful!). Incidentally, if you can see a way to optimize on the first two tests (which are essentially trying to spin up the AJV compiler to test for `unknown keywords`), would be happy to implement as part of this PR (note, calling `.listen()` to invoke the AJV compiler is a bit less than ideal).

Reply:
You can call `fastify.ready()` to do the same thing for those tests.

Reply:
@mcollina Perfect! :)